ui/src/app/store/pod/types.ts

```diff
@@ -70,8 +76,7 @@ export type BasePod = Model & {
   cpu_speed: number;
   created: string;
   default_macvlan_mode: string;
-  // The websocket does not include this value if the pod has no storage pools
-  default_storage_pool?: string;
+  default_storage_pool: string | null;
```

Comment:
This has been fixed on the api side to always return the parameter, but return null as the value if it doesn't exist